submissions/kazeemErinfolami/javascript/Dom/index.js

```diff
@@ -0,0 +1,76 @@
+let qty = document.getElementById('qty'),
+    price = document.getElementById('price'),
+    shippingCost = document.getElementById('shippingCost'),
+    cost = document.getElementById('total'),
+    button = document.getElementById('buyButton');
+
+
+//initial value
+let priceValue = price.innerHTML,
+    costValue = cost.innerHTML;
+
+//RegEx
+const Regex = /^[1-9]/;
+
+function productQuantity() {
+    if (!qty.value.match(Regex)) {
+        button.disable = true
+        button.style = "background-color: grey"
+        qty.nextElementSibling.innerHTML = "⚠️Please enter a valid quantity";
+    } else {
+        button.disabled = false;
+        qty.nextElementSibling.innerHTML = "";
+    }
+}
+function productPrice() {
+    if (qty.value >= 1) {
+        let perPrice = priceValue * qty.value
+        price.innerHTML = perPrice
+    } else if (qty.value === "") {
+        price.innerHTML = priceValue
+        cost.innerHTML = costValue
+    }
+    else {
+        return false
+    }
+}
+
+function shippingPrice() {
+    let perPrice = priceValue * qty.value
+    if (perPrice < 1000) {
+        shippingRange = 100;
```

Comment:
I can't see where `shippingRange` was declared. If it hasn't been declared before kindly use `let` to declare this variable name